lib/load.php

```diff
@@ -100,6 +100,9 @@ function gutenberg_is_experiment_enabled( $name ) {
 	require __DIR__ . '/compat/wordpress-6.2/html-api/class-wp-html-tag-processor.php';
 }
 
+// WordPress 6.2 compat.
+require __DIR__ . '/compat/wordpress-6.3/html-api/class-gutenberg-html-tag-processor-6-3.php';
```

Comment:
this comment seems to contradict the next line.
is this 6.2 compat or 6.3 compat?
or am I just missing something obvious?

Reply:
You're not -- my bad! 😅 I'll update the comment.

Reply:
5b79c179eb